ipwb/replay.py

```diff
@@ -89,16 +89,19 @@ def allowed_file(filename):
 @app.route('/upload', methods=['POST'])
 def upload_file():
     # check if the post request has the file part
+    resp = redirect(request.url)
+    resp.autocorrect_location_header = False
```

Comment:
I think we can avoid the duplication of logic in different places by simply placing `resp.autocorrect_location_header = False` in an `@app.after_request` wrapped function.

Reply:
This means, reverting all these changes and adding this one line in the `setServerHeader` function.

Reply:
@ibnesayeed Have you tested this? Your correction is inconveniently timely.

Reply:
Also, do you foresee any scenarios where this might have ill effects when applied to responses that are not redirects?

Reply:
> Have you tested this? Your correction is inconveniently timely.

No, I have not tested it, but the purpose of `before_request` and `after_request` decorators is to place general logic in one place and keep the code DRY.

> Also, do you foresee any scenarios where this might have ill effects when applied to responses that are not redirects?

No, I don't think it will affect requests that do not have `Location` header in them.